benefits/oauth/views.py

```diff
@@ -0,0 +1,44 @@
+from django.shortcuts import redirect
+from django.urls import reverse
+from django.utils.decorators import decorator_from_middleware
+
+from authlib.integrations.django_client import OAuth
+
+from benefits.core import middleware, session
+
+
+_oauth = OAuth()
+
+
+@decorator_from_middleware(middleware.AgencySessionRequired)
+@decorator_from_middleware(middleware.VerifierSessionRequired)
+def login(request):
+    verifier = session.verifier(request)
+    if not verifier.requires_authentication:
+        return redirect(session.origin(request))
+
+    oauth_client = _oauth.create_client(verifier.auth_provider.name)
```

Comment:
I'm assuming this [verifies the token](https://developers.login.gov/oidc/#token-response), though [the Authlib docs](https://docs.authlib.org/en/latest/client/frameworks.html?highlight=authorize_access_token#parsing-id-token) don't that say that explicitly.

Reply:
Not sure if you meant this comment for another line? This line just creates the client based on the settings.

Reply:
Oops! Indeed.